provisioning/provisioning-service-client-samples/service-link-enrollment-to-ca-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ServiceLinkEnrollmentToCASample.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I was completing step `ii` of the[ .NET implementation](https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/previews/dpsClientCertificates/provisioning/Samples/device/SymmetricKeyWithOperationalCertificateSample#sample) of this feature and though that it could be useful to make it a very small sample. My feelings on it are mixed since its presence without understanding its relationship to the main feature sample in the device client could cause confusion. I've added it into the `pom.xml` for the time being and would love feedback on whether doing this step via a sample like this or just leaving it to the user would be better.